Conductor: further explain setup steps (#117)
packagist/private-packagist-docs: docs/conductor/getting-started.md

To use Conductor:

- You need to be approved for early access to Conductor. [Join the waitlist](/features/conductor) and wait for approval.
- You need a Private Packagist trial or subscription on the cloud plan.
- You need to add a synchronization with your code hosting platform.
- You need to add a workflow to your Continuous Integration platform to run Composer updates.

## Add a synchronization

Once you have a Private Packagist organization and Conductor is enabled for you, log into your Private Packagist organization and head to the "Settings" tab in the main navigation.
Under the "Synchronization" entry you can add [one or more synchronizations](/features/integration-github-bitbucket-gitlab.md) with an organization on your code hosting platform.
This is how you grant us access to your repositories. Once done we will automatically add any repository with a composer.json file in the root directory as a package to Private Packagist.

## Configure Conductor for your packages

Access the "Conductor" tab in the main navigation to see a list of available packages that can be managed via Conductor.

> Each package shown is linked to a repository where you would like to receive pull requests from Conductor. This usually includes all your Composer projects.

Author: Reworded to `where you can receiev`

![Create Authentication Token](/Resources/public/img/docs/conductor/authentication-token.png)

The contents of the variable should look like

```json
{"http-basic": {"repo.packagist.com": {"username": "token", "password": "packagist_out_73a81c..." }}}
```

Conductor needs to verify your CI setup before you can start receiving PRs.

> ## Verify your CI setup

Reviewer: I think there this needs a sentence, that you go to the Conductor tab and click the name of the package of your repository you just added the workflow to.

Author: moved

![Conductor Pull Request](https://packagist.com/img/features/auto-updates/merged-PR-for-a-security-updated.png)

Once you reviewed the changes and merged the PR, Conductor will schedule the next task.      

> If you close the PR, the task will be paused and Conductor will schedule the next task. This is the same effect as using the "Pause" button in the UI. Conductor won't attempt to update the dependency to this exact version again but will schedule updates with newer versions.

Reviewer 2: ```suggestion
If you close the PR, the task will be paused and Conductor will schedule the next task. Clicking the "Pause" button in the UI has the same effect. Conductor won't attempt to update the dependency to this exact version again but it will schedule updates to newer versions.
```
